specification/nhs-website-content-api.yaml

```diff
@@ -32,11 +31,19 @@ info:
       TODO link to API docs
 
     ## API status and roadmap
-    This API has been moved from the nhs.uk developer API website. It is currently in Alpha phase.
+    This API has been moved from the nhs.uk developer API website. It is currently in Alpha phase. If you are an existing user, you need to sign up for the API in the new portal.
```

Comment:
Can this sentence say ...
'If you are an existing user, you need to sign up for the API in the new API Platform.'